catalog/ui/catalog-ui-search/src/test/java/org/codice/ddf/catalog/ui/query/suggestion/UtmUpsCoordinateProcessorTest.java

```diff
@@ -14,65 +14,46 @@
 package org.codice.ddf.catalog.ui.query.suggestion;
 
 import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.closeTo;
 import static org.hamcrest.Matchers.empty;
 import static org.hamcrest.Matchers.hasSize;
-import static org.hamcrest.Matchers.not;
 import static org.hamcrest.core.Is.is;
-import static org.hamcrest.core.IsEqual.equalTo;
-import static org.mockito.Mockito.doReturn;
-import static org.mockito.Mockito.mock;
 
-import com.google.common.collect.ImmutableList;
+import java.util.ArrayList;
 import java.util.LinkedList;
 import java.util.List;
 import org.codice.ddf.spatial.geocoding.Suggestion;
 import org.codice.usng4j.CoordinateSystemTranslator;
-import org.codice.usng4j.DecimalDegreesCoordinate;
-import org.codice.usng4j.UtmUpsCoordinate;
-import org.junit.Before;
+import org.codice.usng4j.impl.CoordinateSystemTranslatorImpl;
 import org.junit.Test;
 
 public class UtmUpsCoordinateProcessorTest {
-  private CoordinateSystemTranslator translatorMock;
 
-  private UtmUpsCoordinateProcessor processor;
+  private CoordinateSystemTranslator translator = new CoordinateSystemTranslatorImpl();
 
-  @Before
-  public void setup() {
-    translatorMock = mock(CoordinateSystemTranslator.class);
-    processor = new UtmUpsCoordinateProcessor(translatorMock);
-  }
+  private UtmUpsCoordinateProcessor processor = new UtmUpsCoordinateProcessor(translator);
 
   @Test
-  public void testUtmStringNorthernHemisphere() throws Exception {
-    setupMocks("13N 234789mE 234789mN", 1.0, 2.0);
+  public void testUtmStringNorthernHemisphere() {
     assertSuggestion(
-        "13N 234789mE 234789mN",
-        "UTM/UPS: [ 13N 234789mE 234789mN ]",
-        ImmutableList.of(new LatLon(1.0, 2.0)));
+        "17R 522908mE 2853543mN", "UTM/UPS: [ 17R 522908mE 2853543mN ]", 25.799891, -80.771484);
```

Comment:
I changed most of the `13N ...` coordinates in the tests to `17R ....` since any coordinates with an N lat band now result in two suggestions